ma/maSolutionTransfer.cc

```diff
@@ -131,7 +131,11 @@ class CavityTransfer : public FieldTransfer
       int bestI = 0;
       for (int i = 0; i < n; ++i)
       {
-        Vector xi = elemInvMaps[i] * point;
+        Vector xi;
+        if (mesh->getShape()->getOrder() == 1)
```

Comment:
the indentation looks odd here - it could be github

Reply:
Fixed this.